src/ui/public/styles/pagination.less

```diff
@@ -42,6 +42,7 @@ paginate {
             text-decoration: none !important;
             font-weight: bold;
             color: @paginate-page-link-active-color;
+            cursor: default;
```

Comment:
Why is this here? one would thin you wouldn't need to set the the value to default.